fdcall_lt_1.12.go

```diff
@@ -0,0 +1,24 @@
+//+build !go1.12,linux
+
+package netlink
+
+import "os"
+
+func setBlockingMode(sysfd int) error {
```

Comment:
I think it'd be nice to add a comment in this file that it exists for compatibility with older versions of Go, although deadlines will be non-functional.